src/Apps/Search/SearchApp.tsx

```diff
@@ -50,7 +49,7 @@ export const SearchApp: React.FC<SearchAppProps> = ({ viewer, children }) => {
   const { query } = location
   const { aggregations } = searchConnection ?? {}
 
-  const term = usePrevious(query.term ?? "")
```

Comment:
`usePrevious` was introduced in https://github.com/artsy/force/pull/12640/files#diff-ee3b179a5ab057d764ecdac38ef50ce5692a5d1ea9ec9de5096fa8199870908aR55 and causes the missing update of the search term.

Reply:
Interestingly the behavior added in #12640 (updating the input when navigating through the dropdown options) does not seem to exist anymore, either before or after this change as far as I can tell.

Wondering about that – but this is a clear improvement over the currently glitchy behavior, so 👍🏽 from me